### ISSUE TITLE
Package biblatex-ieee Error: Failed to update citation style.

### ISSUE THREAD

**Issue by auge** [josephwright/biblatex-ieee#55]
with the latest release of biblatex, using style=ieee fails with above error.

mwe:
```
\documentclass{article}
\usepackage[style=ieee]{biblatex}
\addbibresource{IEEEexample.bib}

\begin{document}
\cite{IEEEexample:biblatex}
\printbibliography
\end{document}
```

debug output:
```
Package biblatex Info: Redefining '\cite'.
Package biblatex Info: Redefining '\parencite'.
Package biblatex Info: Redefining '\footcite'.
Package biblatex Info: Redefining '\footcitetext'.
Package biblatex Info: Redefining '\smartcite'.
Package biblatex Info: Redefining '\supercite'.
Package biblatex Info: Redefining '\textcite'.
Package biblatex Info: Redefining '\textcites'.

[debug] tracing \pretocmd on input line 314
[debug] analyzing '\cbx@textcites@init'
[debug] ++ control sequence is defined
[debug] ++ control sequence is a macro
[debug] ++ control sequence is a macro with parameters
[debug] ++ macro can be retokenized cleanly
[debug] == retokenizing macro now
Package biblatex Info: Redefining '\cites'.
Package biblatex Info: Redefining '\parencites'.
Package biblatex Info: Redefining '\smartcites'.
)
Package biblatex Info: Redefining '\cite'.

[debug] tracing \patchcmd on input line 48
[debug] analyzing '\abx@macro@cite:comp:comp'
[debug] ++ control sequence is defined
[debug] ++ control sequence is a macro
[debug] ++ macro can be retokenized cleanly
[debug] -- search pattern not found in replacement text

! Package biblatex-ieee Error: Failed to update citation style.

See the biblatex-ieee package documentation for explanation.
Type  H <return>  for immediate help.
 ...                                              
                                                  
l.48 ...p,cite:comp:end,cite:comp:inset,cite:dump}
                                                  
Try typing  <return>  to proceed.
If that doesn't work, type  X <return>  to quit.

[debug] tracing \patchcmd on input line 48
[debug] analyzing '\abx@macro@cite:comp:end'
[debug] ++ control sequence is defined
[debug] ++ control sequence is a macro
[debug] ++ macro can be retokenized cleanly
[debug] ++ search pattern found in replacement text
[debug] ++ patching possible
[debug] == retokenizing macro now
[debug] tracing \patchcmd on input line 48
[debug] analyzing '\abx@macro@cite:comp:inset'
[debug] ++ control sequence is defined
[debug] ++ control sequence is a macro
[debug] ++ macro can be retokenized cleanly
[debug] ++ search pattern found in replacement text
[debug] ++ patching possible
[debug] == retokenizing macro now
[debug] tracing \patchcmd on input line 48
[debug] analyzing '\abx@macro@cite:dump'
[debug] ++ control sequence is defined
[debug] ++ control sequence is a macro
[debug] ++ macro can be retokenized cleanly
[debug] ++ search pattern found in replacement text
[debug] ++ patching possible
[debug] == retokenizing macro now
[debug] tracing \patchcmd on input line 52
[debug] analyzing '\abx@macro@cite:dump'
[debug] ++ control sequence is defined
[debug] ++ control sequence is a macro
[debug] ++ macro can be retokenized cleanly
[debug] -- search pattern not found in replacement text

! Package biblatex-ieee Error: Failed to update citation style.

See the biblatex-ieee package documentation for explanation.
```

Hope, this debug may help you.
KR, Benjamin

**Comment by moewew** [comment]
Should be resolved by https://github.com/josephwright/biblatex-ieee/pull/53.

**Comment by auge** [comment]
thanks for the quick response and fix #53 !
the update is already available on CTAN and compilation works.
KR, Benjamin
